electron/constants.ts

```diff
@@ -56,28 +56,32 @@ const isLinux = platform() == 'linux'
 const isFlatpak = Boolean(env.FLATPAK_ID)
 const currentGameConfigVersion: GameConfigVersion = 'v0'
 const currentGlobalConfigVersion: GlobalConfigVersion = 'v0'
+
 const flatPakHome = env.XDG_DATA_HOME?.replace('/data', '') || homedir()
 const home = isFlatpak ? flatPakHome : homedir()
 const configFolder = app.getPath('appData')
-const legendaryConfigPath = `${configFolder}/legendary`
-const heroicFolder = `${configFolder}/heroic`
+const legendaryConfigPath = isLinux
+  ? join(configFolder, 'legendary')
+  : join(home, '.config', 'legendary')
```

Comment:
Shouldn't this be a `isWindows` instead (and then swapping the paths ofc)?  
Where does Legendary keep its config files on Mac?

Reply:
So on linux the configFolder will be .config.
ON windows APPNAME and on Mac Application Support
That why it broke, because I was pointing the legendary config to be always configFolder.

Reply:
but I tested on mac and on windows to check if has fixed and it did. And linux its working normally